spec/components/nav-list/nav-list.automater.js

```diff
@@ -0,0 +1,122 @@
+const dateParse = require('date-fns/parse');
+const isDateEqual = require('date-fns/is_equal');
+const startOfDay = require('date-fns/start_of_day');
+const webdriver = require('selenium-webdriver');
+
+const seleniumProxy = require('../../selenium-proxy');
+
+const By = webdriver.By;
+
+const statuses = {
```

Comment:
Should these be moved to the constants file?

Reply:
good call, done